lib/pages/chat_search/chat_search_view.dart

```diff
@@ -196,74 +201,62 @@ class _SearchItem extends StatelessWidget {
       future: event.fetchSenderUser(),
       builder: (context, snapshot) {
         final user = snapshot.data ?? event.senderFromMemoryOrFallback;
-        return Padding(
+        return Container(
           padding: ChatSearchStyle.itemMargin,
-          child: InkWell(
-            hoverColor: LinagoraRefColors.material().primary[99],
-            borderRadius:
-                BorderRadius.circular(ChatSearchStyle.itemBorderRadius),
+          height: ChatSearchStyle.itemHeight,
```

Comment:
Why not apply style from system?

Reply:
Its applied, check the child of this container

Reply:
IMO: 
- `TwakeListItem` included `height` no need to wrap height outside.
- You can move padding into `TwakeListItem` as well

<img width="433" alt="Screenshot 2024-12-16 at 13 00 52" src="https://github.com/user-attachments/assets/abe6d8c3-c200-48f0-902d-b27bbcc8aebf" />

Reply:
Updated

Reply:
Pls check my comment in your PR

https://github.com/linagora/linagora-design-flutter/pull/54/files#r1887803563